src/Graphics/OpenGLContext/windows/windows_DisplayWindow.cpp

```diff
@@ -298,3 +299,8 @@ void DisplayWindowWindows::_readScreen(void **_pDest, long *_pWidth, long *_pHei
 	glReadPixels(0, m_heightOffset, m_width, m_height, GL_RGB, GL_UNSIGNED_BYTE, *_pDest);
 #endif
 }
+
+graphics::ObjectHandle DisplayWindowWindows::getDefaultFramebuffer()
```

Comment:
_getDefaultFramebuffer, no?

Reply:
Ah yes thanks, I can't compile for Zilmar spec so I did that part blind